@here/harp-mapview/lib/MapView.ts

```diff
@@ -614,6 +614,31 @@ export interface MapViewOptions {
      * @default false.
      */
     synchronousRendering?: boolean;
+
+    /**
+     * Set initial center of the scene in geo coordinates.
+     * Longitude values outside of -180 and +180 are acceptable.
+     * Default is `new GeoCoordinates(25, 0, 30000000)`
```

Comment:
Please divide doc text into 2 sections - typedoc treats first line as `@brief` by default.
1) brief/title / synopsis/summary - one liner
2) Rest, after empty line

2) Please use `@default` in whole patch.